histomicstk/segmentation/label/__init__.py

```diff
@@ -12,6 +12,7 @@
 from .LabelPerimeter import LabelPerimeter
 from .ShuffleLabel import ShuffleLabel
 from .TraceBounds import TraceBounds
+from .trace_boundary import trace_boundary
```

Comment:
I thought you already compared both and showed us that trace_boundary isbf is much faster. 

We should not have two functions which do the same thing in the toolkit. 

If the pure python one is slower, we should just drop it and keep the cython one instead.